django_rq/models.py

```diff
@@ -13,6 +13,7 @@
 class Queue(models.Model):
     """Placeholder model with no database table, but with django admin page
     and contenttype permission"""
+    id = models.AutoField(primary_key=True)
```

Comment:
What's this used for?

Reply:
I removed it. it is causing a Warning, but since it's nor related to this PR I removed it. Thanks.